.circleci/config.yml

```diff
@@ -56,9 +56,8 @@ workflows:
               only: /^v.*/
 
       - architect/integration-test:
-          context: architect
           name: validation-integration-test
-          setup-script: "integration/test/validation/setup.sh"
+          install-app-platform: true
```

Comment:
Removing the setup script. We also don't need the architect context as there are no creds needed for the test.